### PR TITLE
Add Spanish Language

### PR DESCRIPTION
**Pull request by Bedlessly**
For those spanish people who need a spanish version

### CODE REVIEW

File: src/languages/es_ES.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { Language } from "../../lib/util/language";
+import { GuildMember } from "discord.js";
+import { TextChannel } from "discord.js";
+
+export default class extends Language {
+  constructor() {
+    super("en-US", {
```
Comment (GamingGeek) [inline src/languages/es_ES.ts:7]: This need to be changed!

**Review by GamingGeek — CHANGES_REQUESTED** [review]

File: src/languages/es_ES.ts (hunk 1 above)
Comment (Bedlessly) [reply src/languages/es_ES.ts:7]: oopsie!

File: src/languages/es_ES.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import { Language } from "../../lib/util/language";
+import { GuildMember } from "discord.js";
+import { TextChannel } from "discord.js";
+
+export default class extends Language {
+  constructor() {
+    super("es-ES", {
+      language: {
+        DEFAULT: (key: string) =>
+          `${key} aún no se ha localizado para en-US.`,
```
Comment (GamingGeek) [inline src/languages/es_ES.ts:10]: oops forgot to mention this too

**Review by GamingGeek — CHANGES_REQUESTED** [review]

File: src/languages/es_ES.ts (hunk 2 above)
Comment (Bedlessly) [reply src/languages/es_ES.ts:10]: oh shit

**Comment by GamingGeek** [comment]
If you can, please update this regularly.

**Comment by Bedlessly** [comment]
oh uhm, ok but cant 100% gauntee i can

**Pull request merged**